src/Kernel/Point.class.st

```diff
@@ -397,7 +397,20 @@ Point >> isFloatPoint [
 	^ x isFloat and: [ y isFloat ]
 ]
 
-{ #category : #geometry }
+{ #category : #testing }
+Point >> isInRectangle: aRectangle [ 
```

Comment:
I would call it 

 #is**Inside**Rectangle:
 
 to be in alignment with
 
  #is**Inside**Circle:

Reply:
Good suggestion. I will fix this.

Reply:
Thanks!